src/System.Private.ServiceModel/src/System/ServiceModel/Channels/RTSocketConnection.cs

```diff
@@ -190,7 +190,7 @@ protected override void ShutdownCore(TimeSpan timeout)
             catch (ObjectDisposedException objectDisposedException)
             {
                 Exception exceptionToThrow = ConvertObjectDisposedException(objectDisposedException, TransferOperation.Undefined);
-                if (object.ReferenceEquals(exceptionToThrow, objectDisposedException))
```

Comment:
I'll create a resharper coding style config which matches that. As an aside, do you think that would be worth checking in? Resharper supports per solution config files so it would mean anyone loading our solution would get the same coding style automatically. The codeformatter tool is a lot more work to use on an everyday basis, but also does a lot more such as the copyrights.